xapian-core/cluster/euclidian_sim.cc

```diff
@@ -77,29 +78,40 @@ EuclidianDistance::similarity(TermListGroup tlg,
 		wt = tf*idf;
 		docvec2.push_back(make_pair(*titer,wt));
 	}
-	
+
 	vector<pair<string, double> >::iterator ta;
 	vector<pair<string, double> >::iterator tb;
 	double result = 0;
-	
-	// Finding euclidian distance between the above document vectors	
+
+	// Finding euclidian distance between the above document vectors
 	for(ta = docvec1.begin(); ta != docvec1.end(); ta++) {
 		bool found = false;
```

Comment:
This alignment is too deep. Each successive block we move four characters further in, not eight. I think your editor is getting confused between the concept of the tab character (displayed by moving to the next eighth position) and alignment offset (which should be 4).